2023-06-06/keynote/views/index.erb

```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+
+    <title>Ruby Meetup</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <link rel="icon" type="image/x-icon" href="/images/klxrb_slack_logo.png">
```

Comment:
@samgooi4189 : this is the favicon I've sent the screenshot on slack.
Please test any other you prefer.
This isn't meant to be high res on the browser.